src/partials/_hero.hbs

```diff
@@ -0,0 +1,6 @@
+<header class="hero">
+  <div class="hero--container">
+    {{> _AMElogo--color}}
+    <h1 class="h1">{{this.hero}}</h1>
```

Comment:
a class name of  `h1` is an interesting choice. Mind explaining this shift when you are using BEM above.